nucleus/deploy/client.py

```diff
@@ -28,28 +28,53 @@
 class DeployClient:
     """Scale Deploy Python Client extension."""
 
-    def __init__(self, api_key: str, endpoint: str = SCALE_DEPLOY_ENDPOINT):
+    def __init__(
+        self,
+        api_key: str,
+        endpoint: str = SCALE_DEPLOY_ENDPOINT,
+        is_internal: bool = False,
```

Comment:
hmm didn't we say we weren't gonna have an internal / external split so we don't commit this to a public repo?

ok with it i guess